### ISSUE TITLE
Suggestion: loading the dcm using a worker. 

### ISSUE THREAD

**Issue by Betalos** [cornerstonejs/cornerstoneWADOImageLoader#281]
I would like to suggest that the loading of the dicoms be within a worker (sometime loading a series with large number of instances would hang the UI).
I am not sure is this the right place or the Tools repo, since the Tools repos that trigger the loading.
```javascript 
cornerstoneTools.addToolState(element, 'stack', stack); // cornerstone-tools@2.4.0
```

**Comment by silencej** [comment]
I am also working on an issue related to a series with a lot of instances: when the user opens such a series in the first tab, and opens another in the second tab, the second one needs to wait for a long time (we have an API to fetch meta data first from the server, and this API uses the same domain as the image loader XHR, thus it is queued in the very end).
TL; DR: Downloading a large of instances at once causes fetch being blocked.

In nutshell, I'd rather suggest we have a better downloading strategy for the large series.
